nextcloud_install_production.sh

```diff
@@ -395,7 +395,7 @@ occ_command config:system:set upgrade.disable-web --value="true"
 print_text_in_color "$ICyan" "Configuring update notifications specific for this server..."
 download_static_script updatenotification
 check_command chmod +x "$SCRIPTS"/updatenotification.sh
-crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * 6 $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
+crontab -u root -l | { cat; echo "0 $AUT_UPDATES_TIME * * * $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
```

Comment:
Please change pack to 59, we set 59 for a reason, don't remember exactly now, but 0 doesn't work.

Reply:
Are you sure because in the current state, it will get executed at 18:59, no?

Reply:
Yes, I'm sure.

Reply:
this is done 👍